<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorclass</class>
 <widget class="QWidget" name="colorclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a Color</string>
  </property>
  <widget class="QLabel" name="Red">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QSlider" name="redslider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="redbox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QWidget" name="displaywidget" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Green">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QSlider" name="greenslider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="greenbox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="Blue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QSlider" name="blueslider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="bluebox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QWidget" name="chooseBlue" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseRed" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseYellow" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseWhite" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseBlack" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_2" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_3" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_4" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_5" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_6" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_7" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_8" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_9" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_10" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_11" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_12" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="chooseGreen_13" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>redslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bluebox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bluebox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>colorclass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
